js/util/database.js

```diff
@@ -79,4 +80,10 @@ db.open().then(function () {
 
 Dexie.Promise.on('error', function (error) {
   console.warn('database error occured', error)
+
+  if (!dbAlertShown) {
+    window.alert('Make sure no other Min instance is running and then restart Electron.')
```

Comment:
This needs an ```if (window && window.alert)``` around it, since this error could also occur from within a worker that doesn't have a ```window``` object.

Also, could you move the string to the translation file so that it can be translated into other languages? To do that, you would add a line to the ```localization/languages/en-US.json``` file, and then replace the string here with ```l(whatever the string name is)```.

Reply:
Also, for the string itself, it's probably better not to mention Electron, since it's also possible to start multiple instances without trying to develop it. Maybe something like "An error occurred. Please close any other open instances and restart Min"?